src/app/shared/dialog-ministry-plates/dialog-ministry-plates.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, OnInit, ChangeDetectionStrategy, Inject } from '@angular/core';
```

Comment:
Please let catch up on this implementation, no sure why you splitted out the template

Reply:
Please see the exportAs in the ministry-plates.component.ts

Reply:
Closing as I added exportAs in ministry-plates.component.ts
![image](https://user-images.githubusercontent.com/55185857/86338868-87866880-bc5b-11ea-8df9-55c6012c6a75.png)